source/meta.tex

```diff
@@ -374,6 +374,8 @@
   template<class... B> struct conjunction;
   template<class... B> struct disjunction;
   template<class B> struct negation;
+  template<> struct conjunction : true_type { };
+  template<> struct disjunction : false_type { };
```

Comment:
These need indexing, too. Consider contributing towards #5111.

Reply:
They're not really necessary since I added "For the specialization `conjunction<>`, let Bi be `true_type`." Maybe I should remove them again.